### DISCUSSION TITLE
Cannot use operator towhee/to_faiss while using towhee 0.9.0  :-(

### DISCUSSION THREAD

**Discussion by LiuersNick** [towhee-io/towhee#2509] [Q&A] [answered]
I am trying to use Faiss to build an index on the Embedding of images:
```Python
import towhee

img_vectors = (
    towhee.read_csv('info.csv')
      .image_decode['file_name', 'img']()
      .image_text_embedding.clip['img', 'vec'](model_name='clip_vit_base_patch32', modality='image')
      .tensor_normalize['vec','vec']() # normalize vector
      .select['file_name', 'vec']()
)

img_vectors.to_faiss['file_name', 'vec'](findex='./index.bin')
```
but an error occured:
```
ValueError: operator towhee/to-faiss not found!
```

My environements look like this:
- python 3.9
- ipykernel 6.15
- towhee 0.9.0

> p.s. I understand that the operator towhee/to_faiss has been added in version 0.7.1 (it is written in README.md, branch main), but I don’t know why this error is reported

Thanks!

**Comment by junjiejiangjjj** [comment]
@Chiiizzzy   Please take a look

**Comment by Chiiizzzy** [comment, marked as answer]
`to_faiss` is not an operator, it's a mixin. Could you please try to run `from towhee.functional.mixins.faiss import FaissMixin` and post the result here. I tried to reproduce your issue, but it seems things work smoothly in my new conda env.

**Reply by LiuersNick** [reply]
>  I tried to reproduce your issue, but it seems things work smoothly ***in my new conda env.***

Thanks for your reply! I tried to create a new conda env and only install towhee, gradio and faiss-cpu and rerun 
```python
img_vectors.to_faiss['file_name', 'vec'](findex='./index.bin')
```
and this time it works!

I also run the code directly in my notebook 
```python
from towhee.functional.mixins.faiss import FaissMixin
``` 
and  nothing happened.
![image](https://user-images.githubusercontent.com/59303811/233540964-e481a021-bf62-460f-b01d-f3c978314d69.png)

So I guess there may be something wrong with my conda env. I will check it out later by myself.
Thanks again for your reply!
